doc/sphinx/source/data/commondata.rst

```diff
@@ -0,0 +1,355 @@
+a ``dataset_names.yml`` file keeps a mapping of the datasets that were used in 4.0.
+When using the old names in a runcard, ``validphys`` will automatically translate
+them using this file.
+The format of this mapping is as follow:
+
+..  code-block:: yaml
+
+    old_name_1:
+        dataset: new_name_1
+        variant: legacy
+
+                                 
+CommonData Metadata specification
+---------------------------------
+
+The ``metadata.yaml`` file defines unequivocally the datasets implemented within a folder.
+The general structure is a first portion of general information (references, name of the set)
+and a list of ``implemented_observables`` which define separate datasets.
+
+
+Shared information
+==================
+
+
+..  code-block:: yaml
+
+    setname: "EXPERIMENT_PROCESS_ENERGY{_EXTRA}"
+
+    version: 1
+    version_comment: "A comment about this version"
+
+    # References
+    arXiv:
+        url: "https://arxiv.org/abs/XYZ.ABC"
+    iNSPIRE:
+        url: "https://inspirehep.net/literature/XYZ"
+    hepdata:
+        url: "https://www.hepdata.net/record/insXYZ"
+        version: 1
+
+    nnpdf_metadata:
+        nnpdf31_process: "PROCESS"
+        experiment: "EXPERIMENT_NAME"
+
+    implemented_observables:
+      - observable_metadata_1
+      - observable_metadata_2
+
+
+The header of the ``metadata.yaml`` file contains information shared among different datasets.
+
+Setname
+~~~~~~~
+
+Correspond to the name of the set and must be equal to the folder. It acts a s a sanity check.
+
+Versioning
+~~~~~~~~~~
+
+The initial version of a dataset should be set to ``version: 1``.
+Any change on a dataset should be *always* accompanied of a version bump and a ``version_comment`` explaining the update.
+This will allow to keep an exact tracking of all changes to every dataset even if they change over time due to bugs, updates in hepdata, etc.
+
+References
+~~~~~~~~~~
+
+References to the original source of the data. 
+This can be ``arXiv``, ``iNSPIRE`` or ``hepdata``.
+All information must be provided unless it is explicitly missing.
+
+nnpdf_metadata
+~~~~~~~~~~~~~~
+
+Grouping information used internally by ``validphys`` up to NNPDF4.0.
+It accepts the keys ``experiment``, which should in general coincide
+with the ``EXPERIMENT`` key in the ``<setname>`` and the key ``nnpdf31_process``
+which is the process grouping information used in the 3.1 and 4.0 MHOU papers.
+
+Observable specific information
+===============================
+
+Within a ``metadata.yaml`` we can find one or more implemented datasets.
+These correspond to different observables of a single measurement.
+For instance, the LHCB publication of Z rapidity measurements at 13 TeV
+(``setname: LHCB_Z0_13TEV``) contains two observables: Z decay into two electrons
+and Z decay into 2 muons.
+This setname contain two datasets: ``LHCB_Z0_13TEV_DIELECTRON-Y`` and ``LHCB_Z0_13TEV_DIMUON-Y``.
```

Comment:
We should mention in this Section how positivity and integrability are particular in that most of the fields below are not required and should not be defined.

Reply:
We can leave that for next iterations of the documentation. I think we should merge this asap so that the current docs are not heavily outdated.